tests/v2/validation/rbac/rbac_test.go

```diff
@@ -335,6 +338,45 @@ func (rb *RBTestSuite) ValidateAddCMAsProjectOwner() {
 
 }
 
+func (rb *RBTestSuite) ValidateCreateClusterAsResAdmin() {
+	nodeRoles1 := []string{
+		"--etcd",
+		"--controlplane",
+		"--worker",
+	}
+
+	clustersConfig := new(provisioning.Config)
+	config.LoadConfig(provisioning.ConfigurationFileKey, clustersConfig)
+
+	kubernetesVersion := clustersConfig.RKE1KubernetesVersions[0]
+	cni := clustersConfig.CNIs[0]
+	nodeProviders := clustersConfig.NodeProviders[0]
+
+	externalNodeProvider := provisioning.ExternalNodeProviderSetup(nodeProviders)
+	rke1.TestProvisioningRKE1CustomCluster(rb.T(), rb.standardUserClient, externalNodeProvider, nodeRoles1, kubernetesVersion, cni)
+}
```

Comment:
As we've talked offline about this, I don't think this necessarily has to be a method. The only used thing from that suite struct is `testing.T`.  I think this can be a private helper in `rbac.go`.

Reply:
I initially thought having this in the rbac_test.go as there is no much config and looked neat. But after some thought, I moved this to rbac.go based on your suggestion as the extra method seemed redundant here for 2 lines of config. Removed this method and moved the clusterconfig to be obtained from rbac.go which can be used for other validations if need be.